src/main/resources/story/web_app/TrelloBoard.story

```diff
@@ -0,0 +1,49 @@
+Lifecycle:
+Examples:
+/tables/Trello.table
+
+Scenario: initialize variables
+Given I initialize STORY variable `BoardTitle` with value `Title of Board`
+Given I initialize STORY variable `BoardPageTitle` with value `${BoardTitle} | Trello`
+Given I initialize STORY variable `BoardPageUrl` with value `title-of-board`
+Given I initialize STORY variable `ListTitle` with value `List`
+
+Scenario: Log in
+Given I am on the main application page
+When I login with username `<user>` and password `<password>`
+
+Scenario: create Board
+When I create board with `${BoardTitle}`, '${BoardPageTitle}', '${BoardPageUrl}'
+
+Scenario: add list
+When I wait until element located by `By.xpath(//a[@class="open-add-list js-open-add-list"])` appears
+When I click on element located by `By.xpath(//a[@class="open-add-list js-open-add-list"]/span[@class="placeholder"])`
```

Comment:
pls avoid using the elements names in locators until there is a need of doing it. Especially it is about /span and /div elements cause they are just containers and don't make ay sense.
Here instead of //a[@class="open-add-list js-open-add-list"]/span[@class="placeholder"]
better to use
//*[@class="open-add-list js-open-add-list"]/*[@class="placeholder"]

PLease apply for all locators in the TrelloBoard.story file